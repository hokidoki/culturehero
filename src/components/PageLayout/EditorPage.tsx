import React from 'react';
import EditorPageStyle from './EditorPage.styles';
import type EditorPageProps from './EditorPage.types';

export default function EditorPage({ children }: EditorPageProps) {
  return (
    <EditorPageStyle.Layout>
      <EditorPageStyle.EditorArea>{children}</EditorPageStyle.EditorArea>
    </EditorPageStyle.Layout>
  );
}
