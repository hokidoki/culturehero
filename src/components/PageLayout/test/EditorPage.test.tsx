import { fireEvent, render } from '@render';
import EditorPage from '../EditorPage';

describe('EditorPage', () => {
  it('match snapshot ', () => {
    const utils = render(<EditorPage />);
    expect(utils.container).toMatchSnapshot();
  });
});
