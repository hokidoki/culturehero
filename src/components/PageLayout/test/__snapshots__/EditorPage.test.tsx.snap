// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditorPage match snapshot  1`] = `
.c0 {
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
  align-items: center;
  width: 100%;
  min-height: 100%;
  background-color: rgb(17,17,20);
  padding: 40px;
  padding-top: 60px;
}

.c1 {
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
  align-items: flex-start;
  width: 100%;
  max-width: 1000px;
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    />
  </div>
</div>
`;
